src/generic_ui/polymer/app-bar.html

```diff
@@ -0,0 +1,47 @@
+<link rel='import' href='../lib/polymer/polymer.html'>
+<link rel='import' href='../lib/paper-button/paper-button.html'>
+
+<polymer-element name='uproxy-app-bar' attributes='disableback'>
```

Comment:
Seems like maybe just uproxy-back-arrow is a better name?  app-bar makes me think of the whole green bar at the top of uProxy with the hamburger icon the we show above the roster